lang/python/python3/Makefile

```diff
@@ -284,16 +284,6 @@ endef
 Py3Package/python3-light/install:=:
 Package/python3/install:=:
 
-# libuuid is provided by e2fsprogs and uuid/uuid.h is moved into
-# $(STAGING_DIR_HOST)/include/e2fsprogs
-HOST_CFLAGS += \
-	-I$(STAGING_DIR_HOST)/include/e2fsprogs
-HOST_CPPFLAGS += \
-	-I$(STAGING_DIR_HOST)/include/e2fsprogs
-
-HOST_LDFLAGS += \
-	-Wl$(comma)-rpath$(comma)$(STAGING_DIR_HOSTPKG)/lib
```

Comment:
the HOST_LDFLAGS here seem to relate to this 229c23a440192f068376414ae2258fb6b8d9f23b  `python,python3: Fix host Python compilation for macOS`

then previously here 6ade5a1e3adb189d062b707e2b8f731c9faab844 `Python3: Fix host build on OpenSUSE`

and before that, I also sync-ed this from Python2;
but I do remember that linking OpenSSL to Host Python was a bit annoying;

Reply:
Finding OpenSSL should be no problem as of https://github.com/openwrt/openwrt/commit/649e098ec085abad1ac08b545e03db882fbfcc17

Reply:
The CI build log also reports this for host Python:

```
Following modules built successfully but were removed because they could not be imported:
_curses              _curses_panel 
```

A few lines earlier:

```
*** WARNING: renaming "_curses" since importing it failed: /builder/build_dir/hostpkg/Python-3.11.7/build/lib.linux-x86_64-3.11/_curses.cpython-311-x86_64-linux-gnu.so: undefined symbol: extended_color_content
*** WARNING: renaming "_curses_panel" since importing it failed: PyCapsule_Import could not import module "_curses"
```

I'm not sure if this is related to removing `HOST_LDFLAGS`, but https://github.com/openwrt/packages/pull/7129 also mentions an import test failing (although for the _ctypes module instead of _curses).

Reply:
libncurses is also static. I'll look into this.

Reply:
Fixed. Issue is lack of pkgconfig.